src/App.js

```diff
@@ -12,27 +12,19 @@ import Music from "./components/Music/Music";
 import Settigns from "./components/Settings/Settings";
 
 import "./App.css";
```

Comment:
Consider importing component's styles as the last of the imports.

Reply:
done